src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1121,4 +1121,11 @@ protected T computeNext() {
     public static String getDateTimeForDisplay(final ZonedDateTime dateTime) {
         return dateTime.format(longDateTimeFormatter);
     }
+
+    /**
+     * A hack to ensure that numbers are always formatted in the US style.
```

Comment:
Fixed.  (I had just copied over the method from GATK 3 and had preserved the comment.)